.eslintrc

```diff
@@ -107,12 +107,12 @@ rules:
   no-delete-var: 2
   no-label-var: 2
   no-shadow-restricted-names: 2
-  no-shadow: 0
+  no-shadow: 2
   no-undef-init: 2
-  no-undef: 0
+  no-undef: 2
   no-undefined: 0
-  no-unused-vars: 0
-  no-use-before-define: 0
+  no-unused-vars: 2
+  no-use-before-define: 1
```

Comment:
I was having problems with that in a certain file but I'm going to try to go back and see if I can fix it so the rule can be set as error